app/views/site/keyboardMoveHelp.scala

```diff
@@ -27,6 +27,22 @@ object helpModal {
         )
       )
     )
+  def puzzle(implicit ctx: Context) =
+    frag(
+      h2(trans.keyboardShortcuts()),
+      table(
+        tbody(
+          header(trans.navigateMoveTree()),
+          row(frag(kbd("←"), or, kbd("→")), trans.keyMoveBackwardOrForward()),
+          row(frag(kbd("h"), or, kbd("l")), trans.keyMoveBackwardOrForward()),
```

Comment:
So this is really weird, but H, L hotkeys here should be J, K, which are currently paired with up and down arrows (which you would expect to be correct). L is already used for toggling local computer evaluation. Maybe its best for this PR to keep it simple and just make sure what is shown in the keyboard modal is true to the current key bindings, unintuitive as they may be.

Reply:
I see that we now have two rows for J and K now. We should remove the second row.